### DISCUSSION TITLE
R package ospsuite.reportingengine not able to install

### DISCUSSION THREAD

**Discussion by hofferty** [Open-Systems-Pharmacology/Forum#1152] [Models] [answered]
Hi All,

I try to install a package and I face some problems there. I already work with ospsuite and now want to install ospsuite.reportingengine.
After installing the required packages, I can't find it in the library. Find attached my sessioninfo. I suspect, that this package is not working with newer R versions anymore.
Thank you!
Yannick

![sessioninfo](https://user-images.githubusercontent.com/100306143/155544275-0002f968-d8a0-47f2-b033-0d605df601f8.PNG)

**Comment by Yuri05** [comment, marked as answer]
Please close and restart R, remove the previous installation (`remove.packages("ospsuite.reportingengine")`) and reinstall the package
(e.g. `install.packages(pkgs = "C:/ospsuite.reportingengine_X.Y.Z.tar.gz", repos = NULL) ` )
What is the R console output of **install.packages**? Do you get any errors/warnings or does it report "successfully installed"?

**Reply by hofferty** [reply]
Thank you for your answer! When I follow your steps, I get this error if I try to install it.
I get the same warning when I use ( install.packages('./dependencies/OSPSuite.ReportingEngine.zip') )

![error](https://user-images.githubusercontent.com/100306143/155572459-156929a8-0879-4450-ace9-843707b1aacd.PNG)

**Reply by Yuri05** [reply]
Please use the [released version](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/releases/download/v1.2.40/ospsuite.reportingengine_1.2.40.tar.gz) from the [latest package release](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/releases/latest)

**Reply by hofferty** [reply]
It worked when I combined the ospsuite_10.0.72 with the other packages from the released version. For some reason, the ospsuite_10.0.25 from the latest package release created the warning.
Thank you very much!
